ui/webpack.common.ts

```diff
@@ -103,10 +103,6 @@ module.exports = {
       filename: `${STATIC_DIRECTORY}[contenthash:10].css`,
       chunkFilename: `${STATIC_DIRECTORY}[id].[contenthash:10].css`,
     }),
-    new webpack.DllReferencePlugin({
```

Comment:
Why are we removing this?

Reply:
what do you think it does?

Reply:
Based on the docs it looks like it splits the build into different bundles to improve build times.